src/index.css

```diff
@@ -1,13 +1,290 @@
+  transform: translateX(20px);
+}
+
+/* img and its siblings style */
+img{
+  width: 400px;
+  height: 300px;
+  object-fit: cover;
+  border-radius: 20px;
+  background-color: whitesmoke;
+  box-shadow: 2px 0 10px 0 rgba(0,0,0,.521);
+  position: relative;
+  padding: 40px;
+  margin-inline: 20px;
+  margin-block: 10px;
+
+}
+
+
+p{
+  display: inline-block;
+  color: black;
+  font-size: 1.8rem;
+  font-family: 'Indie Flower', cursive;
+}
+
+.itemContainer{
+  display: inline;
+  width: 100vw;
+  position: relative;
+}
+
+.itemContainer p {
+  position: absolute;
+  bottom: 245px;
+  right: 60px;
+  
+}
+
+.buttonContainer{
+  position: absolute;
+  left: 22%;
+  bottom:88% ;
+  opacity: 0.7;
+}
+
+.buttonContainer button{
+  padding: 10px 60px;
+  background-color: antiquewhite;
+  border: none;
+}
+
+.buttonContainer:hover, .buttonContainer:focus {
+  opacity: 1;
+}
+
+figcaption{
+  font-family: 'Indie Flower', cursive;
+  position: absolute;
+  left: 63px;
+  top: -290px;
+  font-size: 1.5rem;
+}
+
+
+/*  shopping cart style*/
+
+.cart{
+  visibility: visible;
+}
+
+nav{
+  background:linear-gradient(to bottom right, #fff,lightgrey);
+  width: 40%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  visibility: hidden;
+  overflow:scroll;
+  box-shadow: 2px 0 10px 0 rgba(0,0,0,.521);
+  z-index: 10;
+}
+
+nav h2{
+  font-size: 2rem;
+}
+
+nav li {
+  max-height: inherit;
+  list-style: none;
+}
+
+nav li img{
+  width:250px;
+  height: 200px;
+  padding: 0;
+}
+
+nav li img:hover, nav li img:focus{
+  transform:scale(1.15);
+}
+
+.crossMark{
+  position: absolute;
+  top: 0;
+  right: 0;
+  cursor: pointer;
+  padding: 10px;
+  margin-right: 15px;
+  display: block;
+  font-size: 2rem;
+  list-style: none;
+}
+
+.crossMark:hover, .crossMark:focus{
+  font-size: 2.8rem;
+  
+}
+
+article{
+  font-size: 1.3rem;
+  font-family: 'Indie Flower', cursive;
+}
+
+nav ul > p{
+  font-family: 'Indie Flower', cursive;
+  padding-top: 50px;
+  color: chocolate;
+  margin-top: 0;
+  text-decoration: underline double 2px;
+  text-underline-offset: 20px;
+}
+
+nav ul button{
+  border: 1px black solid;
+  border-radius: 15px;
+  padding: 5px;
+  font-size: 1.2rem;
+  margin-left: 80px;
+  font-family: 'Indie Flower', cursive;
+
+}
+
+.shoppingCartTitle{
+  margin-top: 60px;
+  margin-bottom: 30px;
+}
+
+
+/* footer style */
+
+footer{
+  margin-top: 50px;
+  background-color: chocolate;
+  padding-block: 10px;
+  opacity: 0.6;
+}
+footer p{
+  font-family:Arial, Helvetica, sans-serif;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+/* filter button style */
+
+form{
+  position: absolute;
+  left: 7vw;
+  top: 300px;
+}
+form h3{
```

Comment:
Maybe style this with a different font,  as well as the "Add to Cart" buttons, as your font styles are so fun everywhere else on the page!